src/_includes/footer.html

```diff
@@ -61,7 +61,7 @@
         }, config.scriptTimeout);
         var d = false;
         var tk = document.createElement('script');
-        tk.src = '//use.typekit.net/' + config.kitId + '.js';
+        tk.src = 'https://use.typekit.net/' + config.kitId + '.js';
```

Comment:
I'd not change this here, and not generally away from what it was.

The browser will pick the right protocol to use.